modules/installation-creating-aws-vpc-subnets-wz.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with compute nodes on AWS Wavelength Zones) 
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-creating-aws-vpc-subnets-wz_{context}"]
+= Creating subnets in Wavelength Zones
+
+Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create the subnets in {zone-type}. Complete the following procedure for each edge zone that you want to deploy compute nodes to.
+
+You can use the provided CloudFormation template and create a CloudFormation stack. You can then use this stack to custom provision a subnet.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS infrastructure, you must review the provided information and manually create the infrastructure. If your cluster does not initialize correctly, you might have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You opted in to the {zone-type} group.
+
+.Procedure
+
+. Go to the section of the documentation named "CloudFormation template for the VPC subnet", and copy the syntax from the template. Save the copied template syntax as a YAML file on your local system. This template describes the VPC that your cluster requires.
+
+. Enter the following command to output the VPC ID value. This value is required for the command that creates a stack of AWS resources. 
++
+[source,terminal]
+----
+VPC_ID=$(aws cloudformation describe-stacks --stack-name "<vpc_stack_name>" \
+  --query 'Stacks[].Outputs[?OutputKey==`VpcId`].OutputValue' --output text)")
+----
+====
+
+. Run the following command to deploy the CloudFormation template, which creates a stack of AWS resources that represent the VPC. You must enter the command on a single line.
```

Comment:
Any reason for this comment/requirement/admonition?
```suggestion
. Run the following command to deploy the CloudFormation template, which creates a stack of AWS resources that represent the VPC. You must enter the command on a single line.
```

Reply:
Hi @mtulio . DO you mean "ou must enter the command on a single line."? Will I remove this sentence?

Reply:
The sentence "You must enter the command on a single line." must be removed